src/platforms/virtual/graphics/display.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Canonical Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License version 2 or 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef MIR_PLATFORMS_VIRTUAL_DISPLAY_H_
+#define MIR_PLATFORMS_VIRTUAL_DISPLAY_H_
+
+#include "platform.h"
+#include <mir/graphics/display.h>
+
+namespace mir
+{
+namespace graphics
+{
+namespace virt
```

Comment:
The reason I didn't do "virtual" is because its a keyword, and I didn't want things to be weird to glance at. Are you still okay with it even though its a keyword?